tests/unit/test_beaker_transformer.py

```diff
@@ -0,0 +1,87 @@
+import pytest
+
+from mrack.providers import providers
+from mrack.providers.beaker import PROVISIONER_KEY as BEAKER
+from mrack.providers.beaker import BeakerProvider
+
+from .mock_data import MockedBeakerTransformer, provisioning_config
+
+
+class TestBeakerTransformer:
+    """Test the Beaker Transformer"""
+
+    domain_name = "example.test"
+    ad_domain_name = "ad.test"
+
+    fedora = {
+        "name": f"fedora.{domain_name}",
+        "role": "client",
+        "group": "client",
```

Comment:
just a nitpick, but for consistency, this should be a `ipaclient` group

Reply:
![come on](https://media.giphy.com/media/MwOuiiTfWfWgM/giphy.gif)